marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/flow/job-query-lib.sjs

```diff
@@ -102,6 +127,9 @@ function buildWhereClause(selectedFacets) {
     if(index < keys.length-1) {
       whereClause = whereClause.concat(" ").concat("AND");
     }
+
+    xdmp.log("WHERE: " + whereClause);
```

Comment:
This log statement needs to be cleaned up. It may be worth converting to a trace statement.